iceoryx_posh/include/iceoryx_posh/internal/roudi/process.hpp

```diff
@@ -42,15 +42,15 @@ class Process
     /// application
     /// @param [in] name of the process; this is equal to the IPC channel name, which is used for communication
     /// @param [in] pid is the host system process id
-    /// @param [in] payloadMemoryManager is a pointer to the payload memory manager for this process
+    /// @param [in] dataSegmentMemoryManager is a pointer to the memory manager of the data segment for this process
```

Comment:
You could consider the name `payloadDataSegmentMemoryManager`. I suppose you did the renaming, to avoid confusing with the new payload term. However, in this context we're at a completely different level. We want to distinguish between the managment segment and the payload data segment.

Reply:
Well, it's not only payload what's stored in that segment but also additional data for the MemPools. To name it `payloadMemoryManager` was wrong in the first place

Reply:
okay, I will rename it to `chunkSegmentMemoryManager`